.github/workflows/core-plugins-build-and-release.yml

```diff
@@ -12,6 +12,7 @@ on:
           - grafana-testdata-datasource
           - parca
           - tempo
+          - zipkin
```

Comment:
FTR, frontend-only plugins are not yet supported in the release process. I have created https://github.com/grafana/grafana/issues/81452 with the missing steps. Can you comment this line for the moment pointing to that issue?

Reply:
please point to the issue in the comment to know how / when to enable it